interface/controller/pokemon.go

```diff
@@ -79,6 +80,60 @@ func (pc *PokemonController) GetPokemon(res http.ResponseWriter, req *http.Reque
 
 }
 
+func (pc *PokemonController) GetPokemonsWorker(res http.ResponseWriter, req *http.Request) {
+	params := req.URL.Query()
+	t := params.Get("type")
+
+	if t == ""{
+		log.Println("ERROR: Empty value in query params type.")
+		http.Error(res, errors.New("empty value in query params type").Error(), http.StatusBadRequest)
```

Comment:
No need to create an error, just to extract the string representation.

I'd recommend to create the error beforehand, and use the `.Error()` method on it, or simply add some terror message text.
```suggestion
		http.Error(res, "empty value in query params type", http.StatusBadRequest)
```